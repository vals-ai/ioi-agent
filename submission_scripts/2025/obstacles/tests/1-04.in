version https://git-lfs.github.com/spec/v1
oid sha256:602e442448dcc16db92986bb6861dd241d778c719c724596b46637b45377421d
size 4778351
