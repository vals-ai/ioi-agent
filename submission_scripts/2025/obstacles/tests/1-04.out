version https://git-lfs.github.com/spec/v1
oid sha256:2aac1b604a7d1d57f83ac6b49d9eabc8f7d5eb838ade60d2a3fc9aadbb24d628
size 400036
